extensions/wikia/AdEngine/js/video/player/porvata/porvataPlayerFactory.js

```diff
@@ -31,6 +31,7 @@ define('ext.wikia.adEngine.video.player.porvata.porvataPlayerFactory', [
 		return {
 			container: prepareVideoAdContainer(videoAdContainer, params),
 			ima: ima,
+			destroyCallbacks: [],
```

Comment:
Can we declare it within scope of `create` method? So it's not accessible on video object